### PR TITLE
[ITBL-5457]-public-cocoapods

### PR DESCRIPTION
**Pull request by tapashmajumder**
1. Create new podspec.
2. Add README.md (more details to be added via another PR).
3. Fix sample apps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+$ pod install
+```
+
+Congratulations! You have now imported the Iterable SDK into your project! 
+
+
+# Using the SDK
+1. Initialize the API with API key. The only required parameter is `apiKey`. You may also like to set default configuration parameters.
+
+	```
```
Comment (vbabenkoru) [inline README.md]: Mark snippets with ```swift to enable syntax highlighting.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+[![CocoaPods](https://img.shields.io/cocoapods/v/Iterable-iOS-SDK.svg?style=flat)](https://cocoapods.org/pods/Iterable-iOS-SDK)
+[![License](https://img.shields.io/cocoapods/l/Iterable-iOS-SDK.svg?style=flat)](https://opensource.org/licenses/MIT)
+[![Docs](https://img.shields.io/cocoapods/metrics/doc-percent/Iterable-iOS-SDK.svg?style=flat)](http://cocoadocs.org/docsets/Iterable-iOS-SDK)
```
Comment (vbabenkoru) [inline README.md]: Cocoadocs are deprecated, so no docs will be generated for this pod. I guess we should remove this until we start hosting the generated docs somewhere.

File: README.md (hunk 1 above)
Comment (tapashmajumder) [reply README.md]: This readme file is thoroughly incomplete. I am working on separate PR to work on documentation.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+You will also need to generate an SSL certificate and private key for use with the push service. See the links at the end of this section for more information on how to do that.
+
+Once you have your APNS certificates set up, go to `Integrations -> Mobile Push` in Iterable. When creating an integration, you will need to pick a name and a platform. The name is entirely up to you; it will be the `appName` when you use `registerToken` in our SDK. The platform can be `APNS` or `APNS_SANDBOX`; these correspond to the production and sandbox platforms. Your application will generate a different token depending on whether it is built using a development certificate or a distribution provisioning profile.
+
+![Creating an integration in Iterable](http://support.iterable.com/hc/en-us/article_attachments/202957719/Screen_Shot_2015-07-30_at_3.15.56_PM.png)
```
Comment (davidtruong) [inline README.md]: Are you going to change the documentation in this PR or a new one to remove the Iterable specific parts.

Comment (tapashmajumder) [reply README.md]: I want to create a separate one for docs. For this one, I just want to fix the podspec etc.

**Comment by tapashmajumder** [comment]
I will be working on docs on a separate PR. Merging this.

**Pull request merged**